### ISSUE TITLE
Allow 1x1 Number Card

### ISSUE THREAD

**Issue by MarkRx** [metabase/metabase#8475]
I want to use Number card to show a single scalar value in as small of a space as possible. Currently it is limited to a minimum of 3x3. Why? All I am showing is a number and 3x3 is wasting valuable space. Given that text cards can be a single square high I see no reason for the arbitrary minimum on a Number card.

**Comment by MarkRx** [comment]
A smaller grid size would also help - #7516

**Comment by mazameli** [comment]
Part of the challenge has to do with what one grid unit equals in terms of pixels at different window sizes. For example, you can see in this mockup with a 1100px-wide browser window, a 1x1 scalar card isn't really feasible (even with the 12pt font used in the mockup of that card). I do think a 2x2 could probably work, though.

![smaller-scalars](https://user-images.githubusercontent.com/2223916/45183408-d76b8400-b1d8-11e8-9aa3-ef14c3b507e9.png)

**Comment by MarkRx** [comment]
Ah yes I was wondering about how scaling was handled after I submitted the issue.

I propose the following:

1. Decrease the number card minimum to be 2 wide and 2 high
2. Decrease the text card minimum width to be 2

**Comment by jornh** [comment]
@MarkRx with your 2x2 minimum this now becomes more or less == https://github.com/metabase/metabase/issues/5662 which asks for < 3x3. 

Your  issue has better content - the other one has more votes.

**Comment by rococode** [comment]
> Part of the challenge has to do with what one grid unit equals in terms of pixels at different window sizes. For example, you can see in this mockup with a 1100px-wide browser window, a 1x1 scalar card isn't really feasible (even with the 12pt font used in the mockup of that card). I do think a 2x2 could probably work, though.

Bumping an older issue here, but my rebuttal to this would be that there's no reason the 1x1 isn't feasible. I don't think anyone is suggesting it to be the default, but if whoever is setting up the dashboard wants more information density, shouldn't they should be free to do so?

In other words, if someone takes the initiative to actually try to drag a card smaller, they should be allowed to do so and if they're unhappy with the smaller size it's trivial for them to revert the change.

In any case, the 1x1 card there is perfectly readable to me - it's far bigger than the default font size of our comments here on GitHub, at least. In a context where the user of the dashboard is familiar with it, I think a smaller font is totally fine.

For example, take this simple mockup:

![](https://i.imgur.com/uqHFebf.png)

The top row is what I currently have on my dashboard. That's a precious 18 grid slots taken up by 2 numbers. The thing is, I know what those numbers are. When I open the dashboard, I know where to look for them - it doesn't matter to me if they're font size 12 or 72, I can read it just fine either way (and reiterating -- if someone is unable to read it at that size, it's not like they're forced to use it). And at 1x1, I can easily space things out however I wish for increased readability (as shown in third row).

The downside of the current limitation is that I have to waste grid space on a bunch of simple numbers, and as a result, do a larger amount of scrolling than I would have to if I could just make the numbers not take up 3 grid slots worth of height each.

I doubt people are going to go out of their way to make a card 1x1 and then say "wow this is so ugly Metabase bad!!" if they aren't happy with the results. They'll just change it back to what it was before.

tl;dr 1x1 may not look good as a default, but can't it at least be an option?

**Comment by tmdoit** [comment]
According to @mazameli response:
> Part of the challenge has to do with what one grid unit equals in terms of pixels at different window sizes. For example, you can see in this mockup with a 1100px-wide browser window, a 1x1 scalar card isn't really feasible (even with the 12pt font used in the mockup of that card). I do think 
a 2x2 could probably work, though.

I agree with @rococode. Technically 1x1 card size is correct as long it displays correct data.
If we take @mazameli example as a use case requirement - 1100 px wide window/screen then using 1x1 card size depends on the use case and consumer preferences.

If an argument is a readability - IMHO it shouldn't because I have to correct card sizes to work for different displays.

**Comment by omerfbaspinar** [comment]
if we have this option , we can use this feature as a narrative title . i mean dynamic titles.

**Comment by smoens** [comment]
I see a lot of advantages to having more flexible selection of number of row/columns and cards that are allowed for tables, cards and markdown. 
Some examples of use cases:
* I want to be able to show the last update timestamp. Having to do this in a card that is 3 rows long restricts place for more content-value cards and graphs. I want this information to be available, but I don't want it to take attention away from the KPI's on which decisions will be based.
* Sometimes I create smaller tables that only contain 1 or 2 columns. 
* I've also used tables as a way to filter content with click behavior, since for some use cases it proved to be more transparent for end users than the available filters on top. Having these types of tables take up 4 columns is unnecessary.

**Comment by chendanli** [comment]
Now text card in dashboard are restricted to 4x1，2x1 will be ideal.

**Comment by johnnyheineken** [comment]
> * I want to be able to show the last update timestamp. Having to do this in a card that is 3 rows long restricts place for more content-value cards and graphs. I want this information to be available, but I don't want it to take attention away from the KPI's on which decisions will be based.

We are just solving the same problem. Encountered it with last update timestamp. This is must have for any report and it is stealing too much space.

**Comment by marksworn** [comment]
How about making the grid squares smaller and using more of them across the width of the page?
That way a 3x3 minimum would still be quite small - but with the opportunity to reduce the unused space.

I would also opt for a setting that doesn't automatically reposition the cards, if that doesn't already exist ))
